Examples/Examples-iOS/IGListKitExamples/Systems/IncrementAnnouncer.swift

```diff
@@ -14,8 +14,7 @@
 
 import UIKit
 
-@objc
-protocol IncrementListener: class {
+@objc protocol IncrementListener: class {
```

Comment:
Does this need to have `@objc` for the single ObjC example we have?

Reply:
It does b/c we're using it in `NSHashTable`, can only put `NSObjectProtocol` stuff in there.